ui/src/components/PalIcon.tsx

```diff
@@ -0,0 +1,146 @@
+import { useMemo } from 'react';
+import _ from 'lodash';
+import { usePals } from '@/state/pals';
+import * as Tooltip from '@radix-ui/react-tooltip';
```

Comment:
seems unused

Reply:
Good catch. I was thinking of using this for on-hover details, but `title` works fine for now.